README.md

```diff
@@ -53,7 +53,7 @@ Let's start by installing the mandatory base of our system:
 
 | RS SW version | Description                                                                                                          | Components Releases |
 | :----------:  | -------------------------------------------------------------------------------------------------------------------- | -------- |
-| **V2.0.0**      | [System Release Note](https://github.com/COPRS/reference-system-software/blob/main/releases/V2.0.0.md)<br>*Scope :<br>\- Deployment of RS infrastructure<br>\- Common Services: Ingestion, Catalog, Compression, Prip...<br>\- L0, L1, L2 products are available for S1, S2 and S3 workflows with SCDF<br>\- Monitoring<br>\- User Web Client* | <br>\- [infrastructure (mandatory)](https://github.com/COPRS/infrastructure/tree/1.6.0)<br>\- [production-common(mandatory)](https://github.com/COPRS/production-common/tree/1.14.0)<br>\- [processing-sentinel-1](https://github.com/COPRS/processing-sentinel-1/tree/1.14.0)<br>\- [processing-sentinel-2](https://github.com/COPRS/processing-sentinel-2/tree/1.8.0)<br>\- [processing-sentinel-3](https://github.com/COPRS/processing-sentinel-3/tree/1.14.0)<br>\- [monitoring](https://github.com/COPRS/monitoring/tree/1.6.0)<br>\- [user-web-client](https://github.com/COPRS/user-web-client/tree/1.10.0) |
+| **V2.1.0**      | [System Release Note](https://github.com/COPRS/reference-system-software/blob/main/releases/V2.0.0.md)<br>*Scope :<br>\- Deployment of RS infrastructure<br>\- Common Services: Ingestion, Catalog, Compression, Prip...<br>\- L0, L1, L2 products are available for S1, S2 and S3 workflows with SCDF<br>\- Monitoring<br>\- User Web Client* | <br>\- [infrastructure (mandatory)](https://github.com/COPRS/infrastructure/tree/1.6.0)<br>\- [production-common(mandatory)](https://github.com/COPRS/production-common/tree/1.15.0)<br>\- [processing-sentinel-1](https://github.com/COPRS/processing-sentinel-1/tree/1.15.0)<br>\- [processing-sentinel-2](https://github.com/COPRS/processing-sentinel-2/tree/1.8.0)<br>\- [processing-sentinel-3](https://github.com/COPRS/processing-sentinel-3/tree/1.15.0)<br>\- [monitoring](https://github.com/COPRS/monitoring/tree/1.6.0)<br>\- [user-web-client](https://github.com/COPRS/user-web-client/tree/1.10.0) |
```

Comment:
why isn't sentinel-2 version upgraded as well as sentinel-1 and sentinel-3?

Reply:
Sentinel-2 is delivered by an other team. S2 is fully independant from productiion-common.